jetty-quickstart/src/main/config/modules/jetty-quickstart.d/quickstart-webapp.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
+
+<Configure class="org.eclipse.jetty.webapp.WebAppContext">
+
+  <Call name="setAttribute">
+    <Arg>org.eclipse.jetty.quickstart.ORIGIN_ATTRIBUTE</Arg>
+    <Arg><Property name="jetty.quickstart.origin" default="origin"/></Arg>
+  </Call>
+
+  <Call name="setAttribute">
+    <Arg>org.eclipse.jetty.quickstart.GENERATE_ORIGIN</Arg>
+    <Arg><Property name="jetty.quickstart.generateOrigin" default="false"/></Arg>
+  </Call>
+  
+  <Call name="setAttribute">
+    <Arg>org.eclipse.jetty.quickstart.QUICKSTART_WEB_XML</Arg>
+    <Arg><Property name="jetty.quickstart.xml" default=""/></Arg>
+  </Call>
+  
+  <Call name="setAttribute">
+    <Arg>org.eclipse.jetty.quickstart.QUICKSTART_MODE</Arg>
+    <Arg>
+      <Call class="org.eclipse.jetty.quickstart.QuickStartConfiguration$Mode" name="valueOf">
+        <Arg><Property name="jetty.quickstart.mode" default="AUTO"/></Arg>
+      </Call>
+    </Arg>
+  </Call>
+  
+  <Call name="addConfiguration">
```

Comment:
Do we still need this?  Didn't we decide that QuickStartConfiguration would be enabled by default?
Do we really need to always have Env, Plus and Annotations for all quickstarted webapps?